staramr/databases/resistance/data/info.ini

```diff
@@ -1,3 +1,4 @@
 [Versions]
 pointfinder_gene_drug_version = 072621.1
 resfinder_gene_drug_version = 072621
+cge_gene_drug_version = 24ee5a0
```

Comment:
This is no longer needed since we aren't storying the CGE gene/drug phenotypes.txt file in the staramr codebase.

Reply:
Good catch. Updated.